.github/workflows/build-test.yml

```diff
@@ -2,13 +2,11 @@ name: Build, Test
 
 on:
   pull_request:
-    branches: ["master"]
     types:
       - opened
-      - edited
       - synchronize
+      - reopened
```

Comment:
these are the standard events

https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request